service-registry/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java

```diff
@@ -216,7 +216,7 @@ public static boolean updateInstanceProperties(Map<String, String> instancePrope
   }
 
   public static Microservice getMicroservice(String microserviceId) {
-    return serviceRegistry.getRemoteMicroservice(microserviceId);
+    return serviceRegistry.getAggregatedRemoteMicroervice(microserviceId);
```

Comment:
not change this

Reply:
if change this... the method org.apache.servicecomb.serviceregistry#Microservice getAggregatedRemoteMicrservice(String microserviceId) will never be invoked? can I delete the method...